setup.py

```diff
@@ -2,8 +2,8 @@
 from sys import version_info
 from setuptools import setup
 
-if version_info < (3, 6):
-    raise ImportError("This module requires Python >=3.6 for asyncio support")
+if version_info < (3, 7):
+    raise ImportError("This module requires Python >=3.7 for asyncio support")
```

Comment:
I'm pretty sure 3.6 supports async def?
"The async def type of coroutine was added in Python 3.5, and is recommended if there is no need to support older Python versions." - https://python.readthedocs.io/en/stable/library/asyncio-task.html

Reply:
All right, let's find out if it passes CI.